### PR TITLE
android support library 28.0.0

### PR DESCRIPTION
**Pull request by dulmandakh**
Bump Android Support Library to 28.0.0. Therefore fix Android CI

Changelog:
----------
[Android] [Changed] - Bump Android Support Library to 28.0.0.

Test Plan:
----------
CI is green

### CODE REVIEW

**Comment by dulmandakh** [comment]
@mdvacca Android CI is green

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK [hunk 1]
```diff
@@ -1,54 +1,131 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v4/BUCK]: Can you add "rn_prebuilt_jar" here?

load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep", "rn_prebuilt_jar")

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK [hunk 2]
```diff
@@ -1,54 +1,131 @@
...
+        ":livedata-core",
+        ":livedata",
+        ":loader",
+        ":localbroadcastmanager",
+        ":print",
+        ":slidingpanelayout",
+        ":swiperefreshlayout",
+        ":versionedparcelable",
+        ":viewpager",
+        react_native_dep("third-party/android/support-annotations:android-support-annotations"),
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v4/BUCK]: let's replace this for:

":android-support-annotations",

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK [hunk 3]
```diff
@@ -1,54 +1,131 @@
...
     ],
 )
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v4/BUCK]: can you add?

fb_native.prebuilt_jar(
    name = "android-support-annotations",
    binary_jar = ":support-annotations-binary.jar",
)

File: ReactAndroid/src/main/third-party/android/support/v4/BUCK [hunk 4]
```diff
@@ -57,60 +134,227 @@ fb_native.android_prebuilt_aar(
...
+
+# remote files
 fb_native.remote_file(
-    name = "android-lifecycle-runtime-aar",
-    sha1 = "95428e5f6bf6875a6a1125d2157c3836a1a837f5",
-    url = "mvn:android.arch.lifecycle:runtime:aar:1.1.0",
+    name = "asynclayoutinflater-aar",
+    sha1 = "3ae7643d120e6da3adbe2d698de923f48c904d1f",
+    url = "mvn:com.android.support:asynclayoutinflater:aar:28.0.0",
 )
```
Comment (mdvacca) [inline ReactAndroid/src/main/third-party/android/support/v4/BUCK]: can you add?

fb_native.remote_file(
    name = "support-annotations-binary.jar",
    sha1 = "ed73f5337a002d1fd24339d5fb08c2c9d9ca60d8",
    url = "mvn:com.android.support:support-annotations:jar:28.0.0",
)

**Pull request closed without merging**